ktor-server/ktor-server-core/jvm/src/io/ktor/server/response/ApplicationResponseFunctionsJvm.kt

```diff
@@ -60,6 +73,14 @@ public suspend fun ApplicationCall.respondFile(file: File, configure: OutgoingCo
     respond(message)
 }
 
+/**
+ * Responds to a client with a contents of a [file]
+ */
+public suspend fun ApplicationCall.respondFile(file: Path, configure: OutgoingContent.() -> Unit = {}) {
```

Comment:
And here `path`, maybe
```suggestion
public suspend fun ApplicationCall.respondFile(path: Path, configure: OutgoingContent.() -> Unit = {}) {
```